### PR TITLE
[APPVEYOR] Enable fast_finish

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
If any errors in building a commit, finish the job for the error commit quickly.
JIRA issue: N/A

### CODE REVIEW

**Comment by SergeGautherie** [comment]
[AppV doc](https://www.appveyor.com/docs/build-configuration/#failing-strategy): 'Sometimes, you want the build fail immediately once one of the job fails'.

> finish the job for the error commit

Does it actually stop the running job only, or all remaining jobs of the build?

Could you explain why you want this change in default behavior?

**Comment by katahiromz** [comment]
> Does it actually stop the running job only, or all remaining jobs of the build?

It stops the remaining jobs.

> Could you explain why you want this change in default behavior?

It becomes less time for waiting for building.

**Comment by SergeGautherie** [comment]
> It becomes less time for waiting for building.

Yes, the choice, which is not mine to make, is between "shorter but first failed job only" and "longer and all job results at once, as in overview".

**Comment by djfkc** [comment]
Suggest that you make it a switch which can be set when running tests.
D Fuller    On Friday, April 19, 2019, 9:42:44 AM CDT, Serge Gautherie <notifications@github.com> wrote:  
 
 

It becomes less time for waiting for building.


Yes, the choice, which is not mine to make, is between "shorter but first failed job only" and "longer and all job results at once, as in overview".

**Comment by GXTX** [comment]
Is this ideal? Now you're unable to see if the issue is an issue with all build types.

File: appveyor.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ version: reactos.appveyor.{build}
 skip_branch_with_pr: true
 clone_depth: 5
 clone_folder: c:\reactos-cov
+matrix:
```
Comment (SergeGautherie) [inline appveyor.yml]: Is the ending space needed?

Comment (katahiromz) [reply appveyor.yml]: Not needed.

**Comment by learn-more** [comment]
> Is this ideal? Now you're unable to see if the issue is an issue with all build types.

Yeah, if there are multiple problems on different builders you have to wait longer now to find that out.

File: appveyor.yml (hunk 1 above)
Comment (SergeGautherie) [reply appveyor.yml]: #1538

**Pull request merged**